docs/api/rsc/matchRSCServerRequest.md

```diff
@@ -11,6 +11,45 @@ unstable: true
 
 Matches the given routes to a Request and returns a RSC Response encoding an `RSCPayload` for consumption by a RSC enabled client router.
 
+The usage of this may differ slightly based on your bundler choice. Here's how it's used with Parcel via `react-server-dom-parcel/server.edge`:
+
+```tsx filename=entry.rsc.ts lines=[10,17-33]
+"use server-entry";
+
+import {
+  createTemporaryReferenceSet,
+  decodeAction,
+  decodeReply,
+  loadServerAction,
+  renderToReadableStream,
+} from "react-server-dom-parcel/server.edge";
+import { unstable_matchRSCServerRequest as matchRSCServerRequest } from "react-router";
+
+import { routes } from "./routes";
+
+import "./entry.browser.tsx";
+
+export function fetchServer(request: Request) {
+  return matchRSCServerRequest({
+    createTemporaryReferenceSet,
+    decodeReply,
+    decodeAction,
+    loadServerAction,
+    request,
+    routes,
```

Comment:
Based on the [template](https://github.com/remix-run/react-router-templates/pull/139/files#diff-b77a93d26634b4a143ea18977a4bfc7a9afc3566c43a33c156cdee0b48fad655R28) I think we should encourage this to be a function, specifically because parcel seems to need it to [magically attach `bootstrapScript`](https://github.com/remix-run/react-router-templates/pull/139/files#diff-b77a93d26634b4a143ea18977a4bfc7a9afc3566c43a33c156cdee0b48fad655R64)

```suggestion
    routes: routes(),
```